classroom.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './student'
+
+class Classroom < Student
+  attr_accessor :label, :student
```

Comment:
- [x] the students instance is not supposed to be edited from the classroom class, which means it's only supposed to be readable from here. Kindly make sure you use the `attr_reader` for the student variable
```suggestion
  attr_accessor :label,
  att_reader :students
```

Reply:
Ohhh I didn't notice it has to be for reading only!